doc-Managing_Infrastructure_and_Inventory/topics/Availability_Zones.adoc

```diff
@@ -2,7 +2,7 @@
 = Availability Zones
 
 An availability zone is a provider-specific method of grouping cloud instances and services.
-{product-title} uses Amazon EC2 regions, OpenStack Nova zones, and Google Compute Engine regions as availability zones. 
+{product-title} uses Amazon EC2 regions and OpenStack Nova zones as availability zones. 
```

Comment:
@dayleparker Not in the scope of this PR, but I see Azure also has availability zones.